src/app/desktop/utils/index.js

```diff
@@ -261,9 +261,19 @@ export const getFilteredVersions = (
   return versions;
 };
 
-export const isLatestJavaDownloaded = async (meta, userData, retry) => {
+export const isLatestJavaDownloaded = async (
+  meta,
+  userData,
+  retry,
+  version = 8
+) => {
   const javaOs = convertOSToJavaFormat(process.platform);
-  const javaMeta = meta.find(v => v.os === javaOs);
+
+  const isJava8 = version === 16;
```

Comment:
wtf

Reply:
Found that too but it’s just the name - the single use still does right

Reply:
change the name..

Reply:
Yeah we will do it